src/mlmc/hdf.py

```diff
@@ -166,26 +172,34 @@ class LevelGroup:
                        "result": {'name': 'collected_values', 'default_shape': (0, 2, 1), 'maxshape': (None, 2, None),
                                   'dtype': np.float64},
                        "time": {'name': 'collected_times', 'default_shape': (0, 2, 1), 'maxshape': (None, 2, None),
-                                'dtype': np.float64}}
+                                'dtype': np.float64},
```

Comment:
I had to comment out this new attribute in order to process results from 01_data. It seems that all samples are skipped.  A conversion or backward compatibility is necessary.